src/components/LeftNav.js

```diff
@@ -3,57 +3,84 @@ import { I18n } from "@aws-amplify/core";
 import Image from "react-bootstrap/lib/Image";
 import DropdownButton from "react-bootstrap/lib/DropdownButton";
 import MenuItem from "react-bootstrap/lib/MenuItem";
-import Glyphicon from "react-bootstrap/lib/Glyphicon";
 import { AiFillCode } from "react-icons/ai";
 import { FaTools, FaHandsHelping } from "react-icons/fa";
 import { IoMdSchool, IoMdCreate } from "react-icons/io";
 import { BiRun } from "react-icons/bi";
 import white from "../assets/Pareto_Lockup-White.png";
-import blue from "../assets/Pareto-Blue-01.png";
-import red from "../assets/Pareto-Red-01.png";
 
 function LeftNav(props) {
   const { chosenLanguage, user, updateState, athletes } = props;
 
-  const textStyle = {
-    color: "rgb(79, 101, 116)",
+  const headingStyle = {
     textDecoration: "none",
-    fontSize: 20,
-    marginLeft: 12,
-    marginTop: 14,
+    fontSize: 15,
+    fontWeight: 600,
+    marginLeft: 16,
+    marginRight: 16,
+    marginTop: 48,
+    paddingTop: 8,
+    textTransform: "uppercase",
+    letterSpacing: "2.5px",
   };
-  const noPadStyle = {
-    color: "rgb(79, 101, 116)",
-    fontSize: 18,
+  const subheadingStyle = {
+    fontSize: 15,
     marginTop: 14,
-    marginLeft: 26,
+    marginLeft: 16,
     display: "flex",
   };
-  const activeTextStyle = {
-    color: "rgb(243, 247, 249)",
+  const activeStyle = {
+    opacity: 1,
     textDecoration: "none",
   };
 
+  const langStyle = {
+    backgroundColor: "black",
+    color: white,
+    padding: "3px",
+  };
+
+  const langTitle = (
+    <Image
+      src={chosenLanguage.image}
+      height="22"
+      width="22"
+      circle
+      style={{ position: "relative", zIndex: "999" }}
+    />
+  );
+
+  // Dropdown styling is very hacky at the moment - will eventually be converted to MUI
   const renderLanguageDropdown = () => (
-    <div style={{ marginLeft: 14 }}>
-      <Image src={chosenLanguage.image} height="26" width="26" circle />
+    <div style={{ display: "flex", alignItems: "center" }}>
       <DropdownButton
```

Comment:
Ticket Item: Replace the DropdownButton npm package with semantic html or MUI menu component/dropdown buttons. https://mui.com/components/menus/

As you can see on line 70, it currently requires a event stopPropagation because the library is meh (my fault here, I don't remember why I event integrated this).

Reply:
Queued - and I agree, this component was surprisingly tough to work with! Excited to swap it out to something else.